src/features/bivariate_manager/components/BivariateMatrixControl/components/QuotientSelector/QuotientSelector.module.css

```diff
@@ -103,21 +97,14 @@
   color: #eee;
   fill: #eee;
 }
-
-.column .denominatorsContainer > div:hover :global(.quotientLabel) {
-  border-top: 24px solid #555;
-}
-
-.row .denominatorsContainer > div:hover :global(.quotientLabel) {
-  border-left: 24px solid #555;
-}
-
 .row .denominatorsContainer > div {
   padding-right: 10px;
-  border-bottom: 1px dotted #999;
 }
 
 .column .denominatorsContainer > div {
   padding-bottom: 10px;
-  border-right: 1px dotted #999;
+}
+
+.selected {
+  background: var(--faint-weak) !important;
```

Comment:
better to avoid `!important` usage, 
nested selector should work better

Reply:
will try to imporve